src/App.svelte

```diff
@@ -51,6 +52,18 @@
     };
   }
 
+  function parseQuery(ctx, next) {
+    queryString = ctx.querystring;
+    queryString = queryString !== "" ? queryString.split("=")[1] : "";
```

Comment:
We eventually should make this more flexible (i.e. break it out into an object of keys and values), but this is good enough for now.